couchbase-server/Dockerfile

```diff
@@ -50,8 +50,9 @@ COPY help.1 /help.1
 
 ARG PROD_VERSION
 ARG STAGING
+ARG TARGETARCH
 ARG CB_RELEASE_URL=http://packages${STAGING}.couchbase.com/releases/${PROD_VERSION}
-ARG CB_PACKAGE=couchbase-server-enterprise-${PROD_VERSION}-rhel8.x86_64.rpm
+ARG CB_PACKAGE=couchbase-server-enterprise-${PROD_VERSION}-rhel8.${TARGETARCH}.rpm
```

Comment:
For arm64/aarch64, we only produce a "linux" rpm/deb, not the OS-specific names. (With the exception of an extra amzn2 one for historical reasons.) We do also produce a "linux" x86_64/amd64 rpm/deb, so it's safe to use that name in both cases here.

Reply:
As discussed last week, I'd like to keep *-linux-* builds earmarked for buildos to keep its builds distinct initially if possible.

Reply:
My point was that there are no "rhel8.aarch64" RPMs, in any release, so this won't work. There are only "linux.aarch64". And starting with 7.2.0 (i hope - no later than 7.5.0) there will only be "linux.x86_64.rpm" installers also.